src/shared/lib/StoreUtils.ts

```diff
@@ -479,21 +479,24 @@ export function mergeMutationsIncludingUncalled(mutationData:MobxPromise<Mutatio
 {
     const idToMutations: {[key: string]: Mutation[]} = {};
 
-    updateIdToMutationsMap(idToMutations, mutationData, generateMutationId);
-    updateIdToMutationsMap(idToMutations, uncalledMutationData, generateMutationId);
+    updateIdToMutationsMap(idToMutations, mutationData, generateMutationId, false);
+    updateIdToMutationsMap(idToMutations, uncalledMutationData, generateMutationId, true);
 
     return Object.keys(idToMutations).map((id:string) => idToMutations[id]);
 }
 
 function updateIdToMutationsMap(idToMutations: {[key: string]: Mutation[]},
                                 mutationData:MobxPromise<Mutation[]>,
-                                generateMutationId:MutationIdGenerator = generateMutationIdByGeneAndProteinChangeAndEvent)
+                                generateMutationId:MutationIdGenerator = generateMutationIdByGeneAndProteinChangeAndEvent,
+                                onlyUpdateIfExists: boolean)
```

Comment:
I can't immediately see `what exists` by just looking at name of the variable? Maybe make it more clear?